bitbots_vision/src/bitbots_vision/vision_modules/color.py

```diff
@@ -406,7 +406,9 @@ def get_static_mask_image(self, optional_image=None):
             mask = self._mask_image(optional_image, _base_color_space)
         else:
             # Mask of default cached image
-            mask = self._mask = self._mask_image(self._image, _base_color_space)
+            mask = self._static_mask
+            if mask is None:  # Check for cached static mask
+                mask = self._mask_image(self._image, _base_color_space)
```

Comment:
This version isn't working ether. The mask would be calculated every time if `self._static_mask` is none because it is not updated.

```suggestion
                mask = self._static_mask = self._mask_image(self._image, _base_color_space)
```